tests/system_tests/test_main.py

```diff
@@ -145,6 +145,7 @@ def test_failure(self):
 
 class DebugTests(TestsBase, unittest.TestCase):
 
+    @unittest.skipUnless(os.environ.get('HAS_NETWORK'), 'no network')
```

Comment:
???  This doesn't need network access does it?

Reply:
No, just localhost apparently. Not sure why I disabled this one - been too long! I'll remove this check and we can see what shakes out.